### DISCUSSION TITLE
Is it possible to have different stroke types based on the series?

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [apexcharts/apexcharts.js#3955] [Q&A]
Is it possible to have different stroke types based on the series? I have 2 series rendering on my chart and want one with solid stroke and the other one with dashed. Something like what is show in the image

<img width="823" alt="CleanShot 2023-08-04 at 20 59 56@2x" src="https://github.com/apexcharts/apexcharts.js/assets/1639302/f6ad74e3-e08f-47ea-a1de-68f4b4bde5e5">

**Comment by easyperl** [comment]
<html>
<body>


stroke: {
   show: true,
   curve: ['straight','straight','straight','straight','smooth'],
   dashArray: [0, 0, 0, 0, 8],
   },


</body>
</html>

This example defines info for 5 series.  All are straight and solid except the 5th being smooth and a dashed line.
This is outlined on the stroke page: https://apexcharts.com/docs/options/stroke/
